include/Webservices/Utils.php

```diff
@@ -123,7 +123,9 @@ function vtws_getUserWebservicesGroups($tabId, $user)
 
 function vtws_getIdComponents($elementid)
 {
-	return explode("x", $elementid);
+	if (strpos($elementid, 'x') !== false) {
+		return explode("x", $elementid);
+	}
 }
```

Comment:
in this case other side was expecting an array. However: is it still needed because of API?